hs_core/tests/api/native/test_copy_resource.py

```diff
@@ -0,0 +1,294 @@
+import os
+import tempfile
+import shutil
+
+from django.contrib.auth.models import Group
+from django.test import TestCase
+from django.core.exceptions import PermissionDenied
+from django.core.files.uploadedfile import UploadedFile
+
+from hs_core import hydroshare
+from hs_core.models import GenericResource
+from hs_core.hydroshare import utils
+from hs_access_control.models import PrivilegeCodes
+from hs_geo_raster_resource.models import RasterResource, OriginalCoverage, CellInformation, \
+    BandInformation
+
+
+class TestCopyResource(TestCase):
```

Comment:
@pkdash Since resource copy and new versioning depend on iRODS for resource file and AVU copy, I don't want to mock any irods operations, hence did not include it in inheritance.